### PR TITLE
Add endpoint to fetch supported NFTs on current chain

### LINKED ISSUE #1791: Add endpoint to fetch supported NFTs on current chain
### Description

Add the following endpoint.

```
HTTP GET /api/v3/nft/supported
RPC  get.nft.supported
```

### Acceptance Criteria

- The endpoints are implemented as expected above
- The response body is up-to-date
- Implement methods in `blockchain-indexer` to fetch this information from the node
- Cache information in connector 
- Update cache on every new block and delete block event
- Add swagger specification
- Update documentation
- Add integration tests
- Add unit tests if applicable

### Additional information
*Response:*

200 OK
```jsonc
{
    "data": {
       "isSupportAllNFTs": false,
       "exactCollectionIDs": ["04000002"],
    },
    "meta": {}
}
```

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?
This PR resolves #1791

### How was it solved?

- [x] The endpoints are implemented as expected above
- [x] The response body is up-to-date
- [x] Implement methods in blockchain-indexer to fetch this information from the node
- [x] Cache information in connector
- [x] Update cache on every new block and delete block event
- [x] Add swagger specification
- [x] Update documentation
- [x] Add integration tests
- [x] Add unit tests if applicable

### How was it tested?
- [x] Local

### CODE REVIEW

File: docs/api/version3.md [hunk 1]
```diff
@@ -2117,6 +2117,49 @@ https://service.lisk.com/api/v3/nft
 ```
 
 
+
+### NFT supported
```
Comment (sameersubudhi) [inline docs/api/version3.md:2121]: Could you please add the relevant entry to the ToC at the top of the file?

File: docs/api/version3.md [hunk 2]
```diff
@@ -2117,6 +2117,49 @@ https://service.lisk.com/api/v3/nft
...
+
+#### Response example
+
+200 OK
+```jsonc
+{
+  "data": {
+    "isSupportAllNFTs": true,
+    "exactCollectionIDs": [
+      "*"
```
Comment (sameersubudhi) [inline docs/api/version3.md:2142]: The example for `exactCollectionIDs` seems incorrect. Please re-verify.

File: services/blockchain-connector/shared/sdk/index.js [hunk 3]
```diff
@@ -117,14 +117,19 @@ const {
...
+	updateCollectionIDsOnBlockChange,
+} = require('./nft');
 
 const init = async () => {
 	// Initialize the local cache
 	await getNodeInfo(true);
 	await cacheRegisteredRewardModule();
 	await cacheFeeConstants();
 	await updateTokenInfo();
+	await updateCollectionIDsOnBlockChange();
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/index.js:132]:
```suggestion
	await updateCollectionIDsOnBlockUpdate();
```

File: services/blockchain-connector/shared/sdk/nft.js [hunk 4]
```diff
@@ -13,13 +13,17 @@
...
+const { Exceptions: { TimeoutException }, Logger, Signals } = require('lisk-service-framework');
 const { timeoutMessage } = require('./client');
 
+const { getNetworkStatus } = require('./network');
+
 const logger = Logger();
 
 let moduleConstants;
 
+let supportedCollectionIDsInfo;
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/nft.js:25]:
```suggestion
let moduleConstants;
let supportedCollectionIDs;
```

File: services/blockchain-connector/shared/sdk/nft.js [hunk 5]
```diff
@@ -37,6 +41,46 @@ const getNFTConstants = async () => {
 	}
 };
 
+// TODO: Remove lint suppression once sdk endpoint is available
+// eslint-disable-next-line no-unused-vars
+const updateCollectionIDs = async (params) => {
+	try {
+		// TODO: Invoke sdk endpoint once available
+		// moduleConstants = await invokeEndpoint('nft_getCollectionIDs',{ chainID: params.chainID });
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/nft.js:49]: Incorrect endpoint. From the current implementation (LS and SDK), looks like it'll only return the collectionIDs for the current chain's collections existing in the NFT sub-store.

This information should actually come from a different endpoint, similar to `token_getSupportedTokens`. Please discuss with the SDK team and create an issue in their repo, if necessary 🙏

File: services/blockchain-connector/shared/sdk/nft.js [hunk 6]
```diff
@@ -37,6 +41,46 @@ const getNFTConstants = async () => {
...
+			throw new TimeoutException('Request timed out when calling \'updateCollectionIDs\'.');
+		}
+		logger.warn(`Error returned when invoking 'nft_getCollectionIDs'.\n${err.stack}`);
+		throw err;
+	}
+};
+
+const getCollectionIDs = async () => supportedCollectionIDsInfo;
+
+const updateCollectionIDsOnBlockChange = async () => {
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/nft.js:64]: Please re-verify if this is really necessary.

File: services/blockchain-indexer/shared/dataService/business/nft/index.js [hunk 7]
```diff
@@ -16,7 +16,10 @@
 const { getNFTs } = require('./nft');
 const { getNFTConstants } = require('./constants');
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/business/nft/index.js:18]:
```suggestion
```

File: services/blockchain-indexer/shared/dataService/nft/index.js [hunk 8]
```diff
@@ -16,7 +16,10 @@
 const { getNFTs } = require('./nft');
 const { getNFTConstants } = require('./constants');
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/nft/index.js:18]:
```suggestion
```

File: services/blockchain-indexer/shared/dataService/business/nft/supported.js [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { Logger } = require('lisk-service-framework');
+const { requestConnector } = require('../../../utils/request');
+const { getNetworkStatus } = require('../network');
+
+const logger = Logger();
+
+const PATTERN_SUPPORT_ALL_NFT_COLLECTIONS = '*';
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/business/nft/supported.js:22]: Constants should go in the constants file.

File: services/blockchain-indexer/shared/dataService/business/index.js [hunk 10]
```diff
@@ -174,6 +178,9 @@ module.exports = {
 	getPosUnlocks,
 	getPosConstants,
 
+	// NFT
+	getNFTSupported,
+
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/business/index.js:183]: Needs to go to L168.

File: services/blockchain-connector/shared/sdk/nft.js [hunk 11]
```diff
@@ -37,6 +41,46 @@ const getNFTConstants = async () => {
...
+	const { chainID } = await getNetworkStatus();
+
+	const updateCollectionIDsListener = async () => {
+		try {
+			await updateCollectionIDs({ chainID });
+		} catch (err) {
+			logger.error(`Error occurred when caching token information:\n${err.stack}`);
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/nft.js:71]:
```suggestion
			logger.error(`Caching supported NFT information failed due to: ${err.message}.`);
			logger.debug(err.stack);
```

File: services/blockchain-indexer/shared/dataService/business/nft/supported.js [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+
+const getNFTSupported = async () => {
+	const nftSupportedResponse = {
+		isSupportAllNFTs: false,
+		exactCollectionIDs: [],
+	};
+
+	try {
+		const { data: { chainID } } = await getNetworkStatus();
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/business/nft/supported.js:31]: Please verify and update if we can cache the processed information.

File: services/blockchain-indexer/tests/unit/shared/dataservice/business/nft/supported.test.js [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+				}),
+			};
+		});
+
+		const { getNFTSupported } = require('../../../../../../shared/dataService/business/nft');
+		const response = await getNFTSupported();
+		expect(response).toEqual({
+			data: {
+				isSupportAllNFTs: true,
+				exactCollectionIDs: ['*'],
```
Comment (sameersubudhi) [inline services/blockchain-indexer/tests/unit/shared/dataservice/business/nft/supported.test.js:53]: Incorrect mocking. Please refer to the supported token implementation.

File: services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+		nftSupportedSchema[this.swaggerApiPath] = { get: {} };
+		nftSupportedSchema[this.swaggerApiPath].get.tags = this.tags;
+		nftSupportedSchema[this.swaggerApiPath].get.summary = 'Requests NFT Module supported collection IDs information.';
+		nftSupportedSchema[this.swaggerApiPath].get.description = getSwaggerDescription({
+			rpcMethod: this.rpcMethod,
+			description: 'Requests supported collection IDs information for the NFT Module.',
+		});
+		nftSupportedSchema[this.swaggerApiPath].get.responses = {
+			200: {
+				description: 'Returns supported collection IDs information for the NFT Modulee.',
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js:37]:
```suggestion
				description: 'Returns supported collection IDs information for the NFT module.',
```

File: services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/nft/supported',
+	rpcMethod: 'get.nft.supported',
+	tags: ['NFT'],
+	get schema() {
+		const nftSupportedSchema = {};
+		nftSupportedSchema[this.swaggerApiPath] = { get: {} };
+		nftSupportedSchema[this.swaggerApiPath].get.tags = this.tags;
+		nftSupportedSchema[this.swaggerApiPath].get.summary = 'Requests NFT Module supported collection IDs information.';
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js:30]: The description might need an update depending on how the implementation changes.
```suggestion
		nftSupportedSchema[this.swaggerApiPath].get.summary = 'Requests NFT module supported collection IDs information.';
```

File: services/gateway/apis/http-version3/swagger/definitions/nft.json [hunk 16]
```diff
@@ -124,5 +124,40 @@
...
+        "properties": {
+            "data": {
+                "description": "NFT module supported collection IDs information.",
+                "type": "object",
+                "required": [
+                    "isSupportAllNFTs",
+                    "exactCollectionIDs"
+                ],
+                "properties": {
+                    "isSupportAllNFTs": {
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/nft.json:143]: Do we not need a pattern specification similar to `patternTokenIDs` as in [`/token/summary`](https://petstore.swagger.io/?url=https://betanet-service.lisk.com/api/v3/spec#/Token/get_token_summary)?

File: services/gateway/sources/version3/mappings/nftSupported.js [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	isSupportAllNFTs: '=,boolean',
+	exactCollectionIDs: 'data.exactCollectionIDs',
```
Comment (sameersubudhi) [inline services/gateway/sources/version3/mappings/nftSupported.js:18]: Please use the standard implementation practice.

File: services/gateway/sources/version3/nftSupported.js [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const nftSupported = require('./mappings/nftSupported');
```
Comment (sameersubudhi) [inline services/gateway/sources/version3/nftSupported.js:16]: Please update the directory structure as discussed offline.

File: tests/integration/api_v3/http/nftSupported.test.js [hunk 19]
```diff
@@ -0,0 +1,39 @@
...
+} = require('../../../schemas/httpGenerics.schema');
+
+const { nftSupportedSchema } = require('../../../schemas/api_v3/nftSupported.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/nft/supported`;
+
+describe('Test NFT Supported API', () => {
+	it('should return NFT module supported collection ids information', async () => {
```
Comment (sameersubudhi) [inline tests/integration/api_v3/http/nftSupported.test.js:30]:
```suggestion
	it('should return NFT module supported collection IDs information', async () => {
```

File: tests/integration/api_v3/rpc/nftSupported.test.js [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+} = require('../../../schemas/rpcGenerics.schema');
+
+const { nftSupportedSchema } = require('../../../schemas/api_v3/nftSupported.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+
+const getNFTSupported = async (params) => request(wsRpcUrl, 'get.nft.supported', params);
+
+describe('get.nft.supported', () => {
+	it('returns NFT module supported', async () => {
```
Comment (sameersubudhi) [inline tests/integration/api_v3/rpc/nftSupported.test.js:31]: Please use consistent testcase descriptions across HTTP/RPC testcases.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: tests/integration/api_v3/http/nftSupported.test.js [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+		const response = await api.get(endpoint);
+		expect(response).toMap(nftSupportedSchema);
+	});
+
+	it('params not supported -> 400 BAD_REQUEST', async () => {
```
[the remark this answers is not in the document]
Comment (priojeetpriyom) [reply tests/integration/api_v3/http/nftSupported.test.js]: Applied.

File: tests/integration/api_v3/http/nftSupported.test.js (hunk 19 above)
Comment (priojeetpriyom) [reply tests/integration/api_v3/http/nftSupported.test.js]: Applied.

File: services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js (hunk 15 above)
Comment (priojeetpriyom) [reply services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js]: Applied.

File: services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js (hunk 14 above)
Comment (priojeetpriyom) [reply services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js]: Applied.

File: services/blockchain-connector/shared/sdk/nft.js (hunk 11 above)
Comment (priojeetpriyom) [reply services/blockchain-connector/shared/sdk/nft.js]: Applied.

File: services/blockchain-indexer/shared/dataService/business/index.js (hunk 10 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/index.js:180]: Applied.

File: services/blockchain-indexer/shared/dataService/nft/index.js (hunk 8 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/nft/index.js]: Applied.

File: services/blockchain-indexer/shared/dataService/business/nft/index.js (hunk 7 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/nft/index.js]: Applied.

File: services/blockchain-connector/shared/sdk/nft.js (hunk 4 above)
Comment (priojeetpriyom) [reply services/blockchain-connector/shared/sdk/nft.js:25]: Applied.

File: docs/api/version3.md (hunk 1 above)
Comment (priojeetpriyom) [reply docs/api/version3.md:2120]: Applied.

File: tests/integration/api_v3/rpc/nftSupported.test.js (hunk 20 above)
Comment (priojeetpriyom) [reply tests/integration/api_v3/rpc/nftSupported.test.js]: Applied.

File: services/gateway/apis/http-version3/swagger/definitions/nft.json (hunk 16 above)
Comment (priojeetpriyom) [reply services/gateway/apis/http-version3/swagger/definitions/nft.json:143]: Applied.

File: services/blockchain-indexer/shared/dataService/business/nft/supported.js (hunk 12 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/nft/supported.js]: We can as the nft supported information is not changing on block update. Applied.

File: services/blockchain-connector/shared/sdk/nft.js (hunk 5 above)
Comment (priojeetpriyom) [reply services/blockchain-connector/shared/sdk/nft.js]: Applied. related sdk issue: https://github.com/LiskHQ/lisk-sdk/issues/8886

File: docs/api/version3.md (hunk 2 above)
Comment (priojeetpriyom) [reply docs/api/version3.md]: Applied.

File: services/blockchain-connector/shared/sdk/nft.js (hunk 6 above)
Comment (priojeetpriyom) [reply services/blockchain-connector/shared/sdk/nft.js]: Applied.

File: services/blockchain-indexer/tests/unit/shared/dataservice/business/nft/supported.test.js (hunk 13 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/tests/unit/shared/dataservice/business/nft/supported.test.js]: Applied.

File: services/blockchain-indexer/methods/dataService/controllers/nft.js [hunk 22]
```diff
@@ -55,7 +55,21 @@ const getNFTConstants = async () => {
 	return constants;
 };
 
+const getNFTSupported = async () => {
```
[the remark this answers is not in the document]
Comment (priojeetpriyom) [reply services/blockchain-indexer/methods/dataService/controllers/nft.js:58]: Applied.

File: services/blockchain-indexer/shared/dataService/business/nft/supported.js (hunk 9 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/nft/supported.js]: Applied.

File: services/gateway/sources/version3/nftSupported.js (hunk 18 above)
Comment (priojeetpriyom) [reply services/gateway/sources/version3/nftSupported.js]: Applied.

File: services/gateway/apis/http-version3/methods/modules/pos/posRewardsLocked.js [hunk 23]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const posRewardsLockedSource = require('../../../../../sources/version3/posRewardsLocked');
+const posRewardsLockedSource = require('../../../../../sources/version3/modules/pos/posRewardsLocked');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/pos/posRewardsLocked.js:16]:
```suggestion
const posRewardsLockedSource = require('../../../../../sources/version3/modules/pos/rewardsLocked');
```

File: services/gateway/apis/http-version3/methods/modules/pos/posStakers.js [hunk 24]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const posStakersSource = require('../../../../../sources/version3/posStakers');
+const posStakersSource = require('../../../../../sources/version3/modules/pos/posStakers');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/pos/posStakers.js:16]:
```suggestion
const posStakersSource = require('../../../../../sources/version3/modules/pos/stakers');
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenAvailableIDs.js [hunk 25]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const tokenAvailableIDsSource = require('../../../../../sources/version3/tokenAvailableIDs');
+const tokenAvailableIDsSource = require('../../../../../sources/version3/modules/token/tokenAvailableIDs');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenAvailableIDs.js:16]:
```suggestion
const tokenAvailableIDsSource = require('../../../../../sources/version3/modules/token/availableIDs');
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenBalances.js [hunk 26]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const tokenBalancesSource = require('../../../../../sources/version3/tokenBalances');
+const tokenBalancesSource = require('../../../../../sources/version3/modules/token/tokenBalances');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenBalances.js:16]:
```suggestion
const tokenBalancesSource = require('../../../../../sources/version3/modules/token/balances');
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenConstants.js [hunk 27]
```diff
@@ -14,7 +14,7 @@
  *
  */
 
-const tokenConstantsSource = require('../../../../../sources/version3/tokenConstants');
+const tokenConstantsSource = require('../../../../../sources/version3/modules/token/tokenConstants');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenConstants.js:17]:
```suggestion
const tokenConstantsSource = require('../../../../../sources/version3/modules/token/constants');
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenSummary.js [hunk 28]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const tokenSummarySource = require('../../../../../sources/version3/tokenSummary');
+const tokenSummarySource = require('../../../../../sources/version3/modules/token/tokenSummary');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenSummary.js:16]:
```suggestion
const tokenSummarySource = require('../../../../../sources/version3/modules/token/summary');
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenTopBalances.js [hunk 29]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const tokenTopBalancesSource = require('../../../../../sources/version3/tokenTopBalances');
+const tokenTopBalancesSource = require('../../../../../sources/version3/modules/token/tokenTopBalances');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenTopBalances.js:16]:
```suggestion
const tokenTopBalancesSource = require('../../../../../sources/version3/modules/token/topBalances');
```

File: services/gateway/apis/http-version3/methods/modules/validators/validatorValidateBLSKey.js [hunk 30]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const validateBLSKeySources = require('../../../../../sources/version3/validatorValidateBLSKey');
+const validateBLSKeySources = require('../../../../../sources/version3/modules/validator/validatorValidateBLSKey');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/validators/validatorValidateBLSKey.js:16]:
```suggestion
const validateBLSKeySources = require('../../../../../sources/version3/modules/validator/validateBLSKey');
```

File: services/gateway/sources/version3/mappings/nftSupported.js
Comment (sameersubudhi) [inline services/gateway/sources/version3/mappings/nftSupported.js:1]: Please rename to `services/gateway/sources/version3/mappings/supported.js` instead.

File: services/gateway/apis/http-version3/swagger/definitions/nft.json [hunk 31]
```diff
@@ -124,5 +124,49 @@
...
+                "required": [
+                    "isSupportAllNFTs",
+                    "patternCollectionIDs",
+                    "exactCollectionIDs"
+                ],
+                "properties": {
+                    "isSupportAllNFTs": {
+                        "type": "boolean",
+                        "example": "false",
+                        "description": "A boolean indicating if all NFTs are supported by the application."
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/nft.json:147]:
```suggestion
                        "description": "A boolean indicating if all NFTs are supported by the application. When true, patternCollectionIDs and exactCollectionIDs will be empty lists."
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenAccountExists.js [hunk 32]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const tokenAccountExistsSource = require('../../../../../sources/version3/tokenAccountExists');
+const tokenAccountExistsSource = require('../../../../../sources/version3/modules/token/tokenAccountExists');
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenAccountExists.js:16]:
```suggestion
const tokenAccountExistsSource = require('../../../../../sources/version3/modules/token/accountExists');
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**